### PR TITLE
Fix usage of hparams and move all mixins to Experiment

### PR DESCRIPTION
**Pull request by Binpord**

### CODE REVIEW

File: src/common/experiment.py [hunk 1]
```diff
@@ -60,6 +64,115 @@ class Experiment(LightningModule):
...
+    def lr_scheduler(self):
+        return self.hparams.get("lr_scheduler", None)
+
+    def train_dataloader(self):
+        if self._train_dataloader is None:
+            if self.dataset == "cifar100":
+                train_data = CIFAR100(
+                    root="data",
+                    train=True,
+                    transform=ToTensor(),
```
Comment (Inkln) [inline src/common/experiment.py:114]: Может еще normalize? Чтобы честно сравниваться с другими экспериментами?

File: src/common/experiment.py [hunk 2]
```diff
@@ -60,6 +64,115 @@ class Experiment(LightningModule):
...
+
+        return self._train_dataloader
+
+    def val_dataloader(self):
+        if self._val_dataloader is None:
+            if self.dataset == "cifar100":
+                val_data = CIFAR100(
+                    root="data",
+                    train=False,
+                    transform=ToTensor(),
```
Comment (Inkln) [inline src/common/experiment.py:134]: Аналогично вопрос про Normalize()

**Review by Inkln — APPROVED** [review]
